client/my-sites/theme/main.jsx

```diff
@@ -9,6 +9,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import page from 'page';
+import { isPremium } from 'my-sites/themes/helpers';
```

Comment:
Ah good point. Since it's already merged tho we can just review that PR and merge it, right? :)